lms/djangoapps/onboarding_survey/models.py

```diff
@@ -5,10 +5,14 @@
 
 from django.contrib.auth.models import User
 from django.db import models
+import django.dispatch
 
 log = logging.getLogger("edx.onboarding_survey")
 
 
+save_interests = django.dispatch.Signal(providing_args=["instance"])
```

Comment:
I think we should define signals in a separate module named `signals.py` instead of `models.py`